src/links/dto/create-link.dto.ts

```diff
@@ -0,0 +1,12 @@
+import { IsUrl, IsUUID } from 'class-validator';
+
+export class CreateLinkDto {
+  @IsUrl()
```

Comment:
**praise**: Using `@IsUrl()` decorator ensures that the `shortUrl` is a valid URL, which is crucial for link management.